packages/components/src/components/ui/Radio.tsx

```diff
@@ -0,0 +1,61 @@
+import { clsx } from "clsx";
+import React from "react";
+import { FieldValues, Path, UseFormRegister } from "react-hook-form";
+
+export function Radio<T extends FieldValues>({
+  name,
+  label,
+  register,
+  initialId,
+  options,
+}: {
+  name: Path<T>;
+  label: string;
+  register: UseFormRegister<T>;
+  initialId: string;
+  options: {
+    id: string;
+    name: string;
+    disabled?: boolean;
+    tooltip?: string;
+  }[];
+}) {
+  return (
+    <div className="flex items-center gap-3">
```

Comment:
I'm kind of curious if we could take a Radio form from some other open Tailwind package, nervous about doing a lot of our own semi-custom stuff. For later though.